README.md

```diff
@@ -2,36 +2,176 @@
 
 [![Build Status](https://travis-ci.org/pglbutt/noodles.svg?branch=master)](https://travis-ci.org/pglbutt/noodles)
 
-The only command line client worthy of pglbutts
+Noodles is a collection of command line tools designed to make your life better. The pglbutt collective currently comprises:
 
-## How
+* spag - __HTTP client__ featuring collections, templates, environments, and history. 
```

Comment:
Can we call it a REST client, maybe? Or something that sort of indicates it's more of an alternative to Postman than to curl.

collections and templates are too generic... I'm thinking like a bullet-point intro answering "why do I want this?":

- define parameterized api requests in YAML files
- easily switch environments to hit different apis
- easily grab previous response data to drive new requests
- list your request history

Or whatever. Maybe it should be a paragraph like what httpie does in its readme. And we need to bold some keywords because people love bold keywords.